src/README.md

```diff
@@ -0,0 +1,52 @@
+# Alfresco JavaScript API
+
+JavaScript client API for the Alfresco REST API and Activiti REST API.
+
+## API clients documentation
+
+- [Authentication Api](src/api/auth-rest-api/README.md)
```

Comment:
@DenysVuika looks like all these paths are incorrect/not working. See screenshot: 
<img width="1357" alt="image" src="https://github.com/Alfresco/alfresco-js-api/assets/115619721/c1eefe48-82ab-4038-8651-8017824e7b48">

It disallows to redirect to these files too when we click on link. 

Probably the reason is that these paths should start with dots like we have in ADF.